### DISCUSSION TITLE
How to merge two user names with same email ids

### DISCUSSION THREAD

**Discussion by rageshhajela16** [community/community#23466] [New to GitHub]
<p>Hi everyone,</p>
<p>I have two usernames associated with the same email id</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/rageshhajela16/">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/rageshhajela16/" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/33627846/181077231-b98ba187-9a7c-434f-a692-a705eb5ae8ce.jpeg" class="thumbnail onebox-avatar" width="184" height="184">

<h3><a href="https://github.com/rageshhajela16/" target="_blank" rel="noopener nofollow ugc">rageshhajela16 - Overview</a></h3>

  <p>Senior AI Engineer and Evangelist. rageshhajela16 has 21 repositories available. Follow their code on GitHub.</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/rageshh">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/rageshh" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/33627846/181077232-c9812f65-7a45-4071-8c8e-67753670eb40.jpeg" class="thumbnail onebox-avatar" width="184" height="184">

<h3><a href="https://github.com/rageshh" target="_blank" rel="noopener nofollow ugc">rageshh - Overview</a></h3>

  <p>rageshh has 2 repositories available. Follow their code on GitHub.</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>Among the above two, I only want the first one. I tried some support blogs available where it talks about merging two accounts, but am unable to proceed further on this.</p>
<p>Can anyone please help me how to merge and transfer the commits as well.</p>
<p>Best Regards,<br>
Ragesh</p>

**Comment by tuves** [comment, marked as answer]
<p>Hi <a class="mention" href="/rageshhajela16">@rageshhajela16</a>! The best place to go for answers to account-specific questions like this one is by opening a ticket on our <a href="https://support.github.com/contact">Support page</a>. Please open a ticket with them :slight_smile:</p>

**Comment by MOHAMED-EHAB-DEV** [comment]
<p>Hi Sir</p>
<p>I don’t understand what do you want but why you create two accounts ?</p>

**Comment by rageshhajela16** [comment]
<p>Yes, your point is valid that why I created two accounts. I don’t even remember now but really want to get rid of one of them, am really sorry :frowning:</p>

**Comment by jfullstackdev** [comment]
<p>yup that’s correct, you can simply add more and more emails in just one account so your commits are tied to just 1 account</p>
<p>hopefully by now you opened a request to fix these two accounts</p>
